pygpsclient/settings_frame.py

```diff
@@ -300,6 +301,26 @@ def _body(self):
             bg=ENTCOL,
             variable=self._mapzoom,
         )
+        self._scatter_scls = [0.1, 1, 5, 10, 15, 20, 25, 50, 100]

```

Comment:
Not really liking the additional slider on the main config panel. Would much prefer the widget to be auto-ranging, or failing that put the slider on the widget itself - perhaps on the bottom left corner.

Reply:
Auto-ranging is not useful for me; this is for looking how a static receiver performs, and I want to be able to zoom in and out. That said, slider on the widget might be good. I was following the example of the map widget (probably obvious given the code's placement). I'll see if I get time to move it later today.

Reply:
Understood - people will need to understand it's not intended as a track follower. 

I think of the web map widget as a 'primary' widget so it seemed sensible to provide the scale slider on the main config panel, but as far as possible I'm trying to keep the 'secondary' widgets as self-contained as possible. It's arbitrary of course.

Reply:
Well, that was easy to move. I think I like it better this way; it's easy to hide the settings panel and still adjust it.